tests/backup/backup_delete_test.go

```diff
@@ -670,3 +671,155 @@ var _ = Describe("{DeleteBucketVerifyCloudBackupMissing}", func() {
 		}
 	})
 })
+
+// DeleteBackupAndCheckIfBucketIsEmpty delete backups and verify if contents are deleted from backup location or not
+var _ = Describe("{DeleteBackupAndCheckIfBucketIsEmpty}", func() {
+	numberOfBackups, _ := strconv.Atoi(getEnv(maxBackupsToBeCreated, "10"))
+	var (
+		scheduledAppContexts     []*scheduler.Context
+		backupLocationUID        string
+		cloudCredUID             string
+		cloudCredUidList         []string
```

Comment:
Looks like cloudCredUidList is not being used anywhere.

Reply:
Ack removed